OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyLogic.cs

```diff
@@ -25,6 +25,7 @@ namespace OpenRA.Mods.Common.Widgets.Logic
 {
 	public class LobbyLogic : ChromeLogic
 	{
+		public const string OpenRALANGame = "OpenRALANGame";
```

Comment:
The logic here really needs to be in the `MasterServerPinger` server trait.  Putting it in the `LobbyLogic` means that games will only be advertised after the first player has joined.  This will also fix the issue where games in progress aren't advertised.